### PR TITLE
Tron 231

### PR DESCRIPTION
**Pull request by EmanekaT**
https://jira.yelpcorp.com/browse/TRON-231
The issue is in function getDuration, the input could be '1 day, 12:02:1.12'  but the getDuration function only handles inputs like '12:02:1.12'.
I tested by setting duration to 1 day, 12:02:1.12, and run make dev, and checked the duration is correct.

### CODE REVIEW

File: tronweb/coffee/views.coffee [hunk 1]
```diff
@@ -34,12 +34,23 @@ window.dateFromNow = (string, defaultString='never') ->
 
 
 window.getDuration = (time) ->
-    [time, ms] = time.split('.')
-    [hours, minutes, seconds] = time.split(':')
-    moment.duration
-        hours: parseInt(hours)
-        minutes: parseInt(minutes)
-        seconds: parseInt(seconds)
+    if time.indexOf("day") != -1
```
Comment (chlgit) [inline tronweb/coffee/views.coffee]: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself  
There are some duplicated codes here. Could you fix that?

**Comment by EmanekaT** [comment]
> If the duration is 10 hours, would it show "0 day 10 hours" or "10 hours"?

Just 10 hours.

File: tronweb/coffee/views.coffee [hunk 2]
```diff
@@ -34,9 +34,14 @@ window.dateFromNow = (string, defaultString='never') ->
 
 
 window.getDuration = (time) ->
+    days = '0'
+    if time.indexOf("day") != -1 or time.indexOf("days") != -1
```
Comment (chlgit) [inline tronweb/coffee/views.coffee]: Should `if time.indexOf("day") != -1` be enough here?

Comment (EmanekaT) [reply tronweb/coffee/views.coffee]: Oh... yes.

**Pull request merged**